pybotvac/robot.py

```diff
@@ -1,15 +1,16 @@
 import hashlib
 import hmac
-import locale
 import os.path
 import re
+import urllib3
 import requests
-import time
+from datetime import datetime, timezone
+from email.utils import format_datetime
 
 from .neato import Neato    # For default Vendor argument
 
 # Disable warning due to SubjectAltNameWarning in certificate
-requests.packages.urllib3.disable_warnings()
+urllib3.disable_warnings(urllib3.exceptions.SubjectAltNameWarning)
```

Comment:
To be honest, these changes are not really part of this PR. Nevertheless, I changed this, because of some errors that occurred during testing.

Reply:
yeah this is fine